bosond.cpp

```diff
@@ -232,6 +276,12 @@ int main(int argc, char** argv) {
                 std::cout << rate << "Hz" << std::endl;
                 t0 = t1;
                 count = 0;
+
+                /* Example of how to retrieve current frame count from camera
+                uint32_t frameCount;
+                roicGetFrameCount(&frameCount);
+                std::cout << frameCount << " frames" << std::endl;
+                */
```

Comment:
is there a readme where this could go? otherwise fine here for now

Reply:
The readme doesn't quite seem like the right place. A developer isn't going to look there for this info.